__tests__/integration/signup-consultant.test.js

```diff
@@ -4,15 +4,35 @@ import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../../src';
 import model from '../../src/models';
+import { hashPassword } from '../../src/utils/passwordHash';
 
-const { User, Consultant } = model;
+const { User, Consultant, AvailableTime } = model;
 
 chai.use(chaiHttp);
 
 describe('Consultant onboarding', async () => {
+  const testConsultant = {
+    uuid: 'b38fcf44-b77f-4149-8d66-454d7a5eacdc',
+    surName: 'Ayooluwaaaa',
+    firstName: 'Olosundeeee',
+    middleName: 'lovisgodrr',
+    email: 'susan.abioya@kodehauz.com',
+    password: hashPassword('Password112'),
+    phone: '07012221112',
+    role: 'consultant',
+    verified: true,
+    gender: 'male',
+    dateOfBirth: '05/06/1994',
+    nationality: 'Nigerian',
+    address: 'No 34B Ewet, Housing Estate',
+    createdAt: new Date(),
+    updatedAt: new Date()
+  };
   before(async () => {
-    User.destroy({ where: {}, force: true });
-    Consultant.destroy({ where: {}, force: true });
+    await User.destroy({ where: {}, force: true });
+    await User.create(testConsultant);
+    await Consultant.destroy({ where: {}, force: true });
+    await AvailableTime.destroy({ where: {}, force: true });
   });
 
```

Comment:
Since running our tests will undo migrations and automatically remove previous data from the test database. I think we should only remove specific data from a relation. Instead of removing all. It would affect my tests for authentication which requires testing some data that is already in the database.